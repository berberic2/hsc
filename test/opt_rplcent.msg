test/simple.hsc (7,7): Note 46: replaced `ü' by `&uuml;'
test/simple.hsc (7,9): Note 46: replaced `ß' by `&szlig;'
test/simple.hsc (7,29): Note 46: replaced `ö' by `&ouml;'
test/simple.hsc (7,40): Note 46: replaced `Ü' by `&Uuml;'
test/simple.hsc (7,43): Note 46: replaced `ä' by `&auml;'
test/simple.hsc (7,46): Note 46: replaced `ä' by `&auml;'
test/simple.hsc (9,34): Note 46: replaced `ü' by `&uuml;'
