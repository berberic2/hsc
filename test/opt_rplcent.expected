<HTML><HEAD><TITLE>Simple test</TITLE></HEAD>
<BODY>just a &quot;simple" test with no special
features &amp; &lt;tags&gt;;.

And now something German (sort of):

Nur f&uuml;r&szlig; Te&szlig;ten der d&ouml;&ouml;fen &Uuml;ml&auml;ut&auml;.

And numeric : &#229;bla &#xE5; s&uuml;lz&#Xe5;
Beyond 8-bit: &#1048; fasel &#x6C34; laber

</BODY></HTML>
